app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -78,6 +85,15 @@ def available_images
       client.images
     end
 
+    def create_disk(size, image_uuid)
+      client.disks.create(
+        :name         => "foreman-#{Time.now.to_i}",
```

Comment:
shouldn't this use the VM name?  We use Time.now in vm_instance_defaults, but it's overwritten somewhere else with the real VM name later on in the process - this doesn't look like it is.